.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,26 @@
+name: Node.js CI
+
+on: [pull_request]
```

Comment:
This should include push as well.

Reply:
@mcollina on: [push, pull_request] will create redundant builds in the Github Actions. The result will be that every time a push is made on a PR, instead of just the tests running once they will all be run twice. See for instance: https://github.com/mqttjs/MQTT.js/actions/runs/44644983